use failure::Error;
pub fn run(contents: &[Vec<String>]) -> Result<(), Error> {
    Ok(())
}
